version.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
   "version": "2.7",
-  "versionHeightOffset": -1,
+  "versionHeightOffset": 180,
```

Comment:
@Martin, why isn't the branch based on `release/2.7.171` instead of `2.7.170` ? Hard-coding of `180` value seem to be a necessary workaround because of that.

Reply:
we just focus on the least amount of necessary changes for this hotfix, just enable the batching per environment